docs/src/components/ComponentDoc/BehaviorDescription.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { Loader } from '@stardust-ui/react/src'
+
+const AccessibilityDescription = React.lazy(() => import('./AccessibilityDescription'))
+
+const item = '- '
+
+const BehaviorDescription: React.FunctionComponent<{ value: string }> = ({ value }) => {
+  // doctrine has a bug where it ignores list item indicator (-) if it is in the beginning of the comment
+  // because of that, add the list item indicators after parsing
```

Comment:
Broken parser sucks 👎 

I tried to inspect and found that `doctrine` parses only `@example` correctly:
https://github.com/eslint/doctrine/blob/master/lib/doctrine.js#L682

I also tried [`comment-parser`](https://github.com/yavorskiy/comment-parser) that currently is used in [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc), but there is the same issue even on `@example` 💣